File: src/main/java/com/movierental/Customer.java
package com.movierental;

public class Customer {
    private String name;
    private Rentals rentals = new Rentals();

    public Customer(String name) {
        this.name = name;
    }

    public void addRental(Rental arg) {
        rentals.add(arg);
    }

    public String getName() {
        return name;
    }

    public String Textstatement() {
        return new TextStatement().display(this.rentals, this.getName());
    }

    public String Htmlstatement() {
        return new HtmlStatement().display(this.rentals, this.getName());
    }
}

File: src/main/kotlin/com/cleancode/kotlin/Customer.kt
package com.cleancode.kotlin

data class Customer(val name: String) {
    var rentals = Rentals()

    fun addRental(rental: Rental) {
        rentals.add(rental)
    }

    fun Textstatement(): String {
        return TextStatement().display(rentals, name)
    }

    fun Htmlstatement(): String {
        return HtmlStatement().display(rentals, name)
    }

}